### PR TITLE
Updated dependencies and build chain

### PR DESCRIPTION
**Pull request by lorem--ipsum**
Also, switched to jest, and now using `esModuleInterop` because that's what all the cool kids on the block do.

### CODE REVIEW

File: src/date-parser/date-parser.spec.ts [hunk 1]
```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2014-2015 Metamarkets Group Inc.
+ * Copyright 2015-2016 Imply Data, Inc.
```
Comment (vogievetsky) [inline src/date-parser/date-parser.spec.ts]: could you update all the `-2016` to `-2019`

File: package.json [hunk 2]
```diff
@@ -23,41 +23,45 @@
...
-    "blanket": "1.2.3",
-    "browserify": "16.2.2",
-    "chai": "4.1.2",
-    "gulp": "3.9.1",
-    "gulp-mocha": "6.0.0",
-    "gulp-typescript": "5.0.0-alpha.3",
+    "@types/jest": "^24.0.15",
+    "@types/jest-expect-message": "^1.0.0",
+    "@types/moment-timezone": "^0.5.12",
+    "@types/node": "^12.0.8",
```
Comment (vogievetsky) [inline package.json:64]: could you add awesome-code style for linting also please

File: src/date-parser/date-parser.spec.ts (hunk 1 above)
Comment (vogievetsky) [inline src/date-parser/date-parser.spec.ts]: could you update all the `-2016` to `-2019`

Comment (lorem--ipsum) [reply src/date-parser/date-parser.spec.ts]: 👍

File: package.json (hunk 2 above)
Comment (lorem--ipsum) [reply package.json:64]: 👍

Comment (vogievetsky) [inline package.json:64]: could you add awesome-code style for linting also please

**Comment by evasomething** [comment]
💧

**Pull request merged**